docs/examples/use_cases/paddle/ssd/train.py

```diff
@@ -23,94 +23,75 @@
-            output_layout=types.NCHW,
-            pad_output=False)
-
-        self.rng1 = ops.Uniform(range=[0.5, 1.5])
-        self.rng2 = ops.Uniform(range=[0.875, 1.125])
-        self.rng3 = ops.Uniform(range=[-0.5, 0.5])
-        self.coin = ops.CoinFlip(probability=0.5)
-        self.build()
-
-    def define_graph(self):
-        saturation = self.rng1()
-        contrast = self.rng1()
-        brightness = self.rng2()
-        hue = self.rng3()
-        flip = self.coin()
-
-        images, bboxes, labels = self.reader(name="Reader")
-
-        crop_begin, crop_size, bboxes, labels = self.crop(bboxes, labels)
-        bboxes = self.bbflip(bboxes, horizontal=flip)
-
-        images = self.roi_decode(images, crop_begin, crop_size)
-        images = self.resize(images)
-        images = self.hsv(images.gpu(), hue=hue, saturation=saturation)
-        images = self.bc(images, brightness=brightness, contrast=contrast)
-        images = self.cmnp(images, mirror=flip)
-        return images, bboxes, labels
+def create_coco_pipeline(file_root,
+                         annotations_file,
+                         batch_size=1,
+                         device_id=0,
+                         num_threads=4,
+                         local_rank=0,
+                         world_size=1):
+    pipeline = Pipeline(batch_size, num_threads,
+                        local_rank, seed=42 + device_id)
+
+    with pipeline:
+        images, bboxes, labels = fn.coco_reader(file_root=file_root,
+                                                annotations_file=annotations_file,
+                                                skip_empty=True,
+                                                shard_id=local_rank,
+                                                num_shards=world_size,
+                                                ratio=True,
+                                                ltrb=True,
+                                                random_shuffle=False,
+                                                shuffle_after_epoch=True,
+                                                name="Reader")
+
+        crop_begin, crop_size, bboxes, labels = fn.random_bbox_crop(bboxes, labels,
+                                                                    device="cpu",
+                                                                    aspect_ratio=[0.5, 2.0],
+                                                                    thresholds=[0, 0.1, 0.3, 0.5, 0.7, 0.9],
+                                                                    scaling=[0.3, 1.0],
+                                                                    bbox_layout="xyXY",
+                                                                    allow_no_crop=True,
+                                                                    num_attempts=1)
+        images = fn.image_decoder_slice(
+            images, crop_begin, crop_size, device="cpu", output_type=types.RGB)
```

Comment:
before there was ```device='mixed'```. Did you change it in purpose?

Reply:
Done